nexus/src/external_api/params.rs

```diff
@@ -250,6 +251,12 @@ impl Into<ByteCount> for BlockSize {
     }
 }
 
+impl Into<u64> for BlockSize {
```

Comment:
Any reason this isn't a `From<BlockSize> for u64`?  The docs [generally recommend that instead](https://doc.rust-lang.org/std/convert/trait.Into.html).

Reply:
I didn't know that's what the docs recommend, and it unfortunately doesn't help. In this case, I can't do a From, it has to be a TryFrom to reject invalid block sizes, and Rust doesn't seem to be deriving the appropriate TryInto.

Reply:
It looks to me like any valid `BlockSize` is a valid `u64`?  If it weren't, I don't think you could impl `Into<u64> for BlockSize` here.

Reply:
You're right, I misinterpreted what I needed to do. Changed in 5b35c58e